### PR TITLE
Elm cms

### PR DESCRIPTION
**Pull request by frismaury**
This will be the CMS, for now only have the dashboard data placeholders.

### CODE REVIEW

File: cms/Commands.elm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+showDashboard : Cmd Msg
+showDashboard =
+    Http.get fetchConferencesUrl conferencesDecoder
+        |> RemoteData.sendRequest
+        |> Cmd.map Msgs.OnFetchConferences
+
+
+
+-- fetch users?
```
Comment (PedroAlbR) [inline cms/Commands.elm]: This comment is not neccesary

File: cms/View.elm [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import Html exposing (Html, div, text)
+import Models exposing (Model)
+import Msgs exposing (Msg)
+import Pages.Dashboard
+
+
+--view : Model -> Html Msg
+--view model =
+--    div []
+--        [ text model ]
```
Comment (PedroAlbR) [inline cms/View.elm]: This is not needed imo

File: elm-app/Pages/Aboutus/Aboutus.elm [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import Html exposing (..)
+import Msgs exposing (Msg)
+import Components.Base as Base
+
+
+init : Html Msg
+init =
+  Base.content
+    [ text "About us Page"
+    , h1 [] [ text "testing" ]
```
Comment (PedroAlbR) [inline elm-app/Pages/Aboutus/Aboutus.elm]: I guess you forgot to delete this file :rofl:

File: elm-app/elm-package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
     "repository": "https://github.com/user/project.git",
     "license": "BSD3",
     "source-directories": [
-        "."
+        ".",
+        "./Pages/"
```
Comment (PedroAlbR) [inline elm-app/elm-package.json]: Care to explain this pls?

File: elm-package.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    "source-directories": [
+        "."
+    ],
+    "exposed-modules": [],
+    "dependencies": {
+        "elm-lang/core": "5.1.1 <= v < 6.0.0",
+        "elm-lang/html": "2.0.0 <= v < 3.0.0"
+    },
+    "elm-version": "0.18.0 <= v < 0.19.0"
+}
```
Comment (PedroAlbR) [inline elm-package.json]: This file is in the root, there is no app here.

File: cms/package.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "api-project",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "webpack",
+    "watch": "webpack --watch",
+    "dev": "webpack-dev-server --port 8000 --host 0.0.0.0"
```
Comment (PedroAlbR) [inline cms/package.json]: We need to change this to another port because this one is used by the api admin port.

Comment (frismaury) [inline cms/package.json]: So, we need to change that: the client app: 8080, the cms: 8000 and the Go api: 8081
:grimacing:

Comment (PedroAlbR) [inline cms/package.json]: Or change this to the :8082

Comment (frismaury) [inline cms/package.json]: oks

**Pull request merged**